SLiMgui/SLiMHelpClasses.rtf

```diff
@@ -5617,18 +5617,29 @@ Normally, the full SLiM script used to generate the tree sequence is written out
 \pard\pardeftab397\li720\fi-446\ri720\sb180\sa60\partightenfactor0
 
 \f3\fs18 \cf2 \expnd0\expndtw0\kerning0
-\'96\'a0(void)treeSeqRememberIndividuals(object<Individual>\'a0individuals)\
+\'96\'a0(void)treeSeqRememberIndividuals(object<Individual>\'a0individuals, [logical$\'a0permanent\'a0=\'a0T])\
```

Comment:
All these RTF diffs will need to be redone, because the RTF files (and HTML files in QtSLiM, too) are generated from the original doc, which is a macOS Pages document.  Sorry for the hassle, but can you just send me some kind of before/after comparison so I can apply the necessary changes to the Pages document?  And then once you've done that you should remove all RTF/HTML files from this PR; I'll push the doc changes separately, I suppose, since that seems simplest.

Reply:
Ah, right. Or if pages has a "track changes" thing, and I can access the original doc, I could change that and send it back?

Reply:
Ah, if you're on a Mac that would work fine.  Pages does have a change-tracking feature.  It's a big doc.  I'll send it to you through Slack, shall I?

Reply:
Sure. Slack would be fine, or WeTransfer or something would work too

Reply:
Just sent it to you on Slack.  :->

Reply:
Thanks - just sent back with my changes tracked. I also just edited this commit to remove the RTF file changes, which I guess you'll commit separately from your master Pages doc. Do you want me to squash all the commits together into one for this PR @bhaller , or do you prefer them to be split up into pieces, like the 5 at the moment?

Reply:
OK @hyanwong.  I see your Slack message.  How about this.  I will integrate your changes into the manual, and I'll Slack you back the .rtf and .html files with those diffs incorporated.  You can replace the existing files with those files in your PR, and then squash down the PR, and then we'll merge it, once it's ready of course.  This will be fine as long as I don't touch those files again on my side until we merge the PR, and that's fine since it looks like the PR will be mergeable quite soon now.  Sound good?